themes/src/main/resources/theme/base/admin/resources/partials/client-detail.html

```diff
@@ -37,7 +37,7 @@
                 </div>
                 <kc-tooltip>{{:: 'client.enabled.tooltip' | translate}}</kc-tooltip>
             </div>
-            <div class="form-group clearfix block">
+            <div class="form-group clearfix block" data-ng-show="protocol != 'docker-v2'">
                 <label class="col-md-2 control-label" for="consentRequired">{{:: 'consent-required' | translate}}</label>
                 <div class="col-sm-6">
                     <input ng-model="clientEdit.consentRequired" name="consentRequired" id="consentRequired" onoffswitch on-text="{{:: 'onText' | translate}}" off-text="{{:: 'offText' | translate}}"/>
```

Comment:
Is there a way to sort Client Protocol list for #protocol select input in such a way that *docker-v2* is the last one in the list - it makes more sense for it to be the last choice as it's reasonable to expect that it will be the least used option?